apps/files/src/services/DropService.ts

```diff
@@ -29,9 +29,12 @@ import { getUploader } from '@nextcloud/upload'
 import { joinPaths } from '@nextcloud/paths'
 import { showError } from '@nextcloud/dialogs'
 import { translate as t } from '@nextcloud/l10n'
+import { loadState } from '@nextcloud/initial-state'
 
 import logger from '../logger.js'
 
+const forbiddenCharacters = loadState('files', 'forbiddenCharacters', '') as string
```

Comment:
```suggestion
const forbiddenCharacters = loadState<string>('files', 'forbiddenCharacters', '')
```

No explicit cast needed